appveyor.yml

```diff
@@ -3,7 +3,7 @@ pull_requests:
   do_not_increment_build_number: true
 image: Visual Studio 2017
 configuration: Release
-platform: Any CPU
+#platform: Any CPU

```

Comment:
If we set platform setting, AppVeyor cannot resolve path.
https://ci.appveyor.com/project/budougumi0617/testable/build/0.0.60-civmrlfx
```
msbuild "C:\projects\testable\Testable.sln" /verbosity:minimal /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
Microsoft (R) Build Engine version 15.3.409.57025 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.
  Testable -> C:\projects\testable\Testable\bin\Release\netstandard1.6\Testable.dll
TargetClass.cs(39,20): warning CS0414: The field 'TargetClass.privateStaticInt' is assigned but its value is never used [C:\projects\testable\Testable.Tests\Testable.Tests.csproj]
  Testable.Tests -> C:\projects\testable\Testable.Tests\bin\Release\netcoreapp1.1\Testable.Tests.dll
dotnet pack "C:\projects\testable\Testable\Testable.csproj" --configuration Release --output "C:\Users\appveyor\AppData\Local\Temp\1\g4d535j57e" --no-build
Microsoft (R) Build Engine version 15.3.409.57025 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.
C:\Program Files\dotnet\sdk\2.0.0\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(141,5): error : The file 'C:\projects\testable\Testable\bin\Any CPU\Release\netstandard1.6\Testable.dll' to be packed was not found on disk. 
```